v2/index.html

```diff
@@ -324,13 +331,20 @@ <h3>Key Signature</h3>
         <div class="issue" data-number="73"></div>
         <h3>Time Signature</h3>
         <p>
-            An encoding MAY include a time signature. There are two main categories of time signature forms,
-            <abbr title="Common Western Music Notation">CWMN</abbr> and Mensural. If a mensuration sign is
-            specified, the <a>clef</a> MUST specify a <code>+</code> separator to indicate the encoding is in mensural
-            notation.
+            An encoding MAY include a time signature.
         </p>
         <p>
-            Encodings MUST NOT mix CWMN and Mensural time signatures.
+            There are two main categories of time signature forms, <a>Common Western Music Notation</a>
+            (<abbr title="Common Western Music Notation">CWMN</abbr>) and Mensural.
+            If a mensuration sign is specified, the <a>clef</a> MUST specify a <code>+</code> separator to
+            indicate the encoding is in mensural notation.
+        </p>
+        <p>
+            Encodings MUST NOT mix <abbr title="Common Western Music Notation">CWMN</abbr> and Mensural
+            time signatures.
+        </p>
+        <p>
+            If an encoding omits a time signature, the encoding MUST be interpreted as <abbr>CWMN</abbr>.
```

Comment:
I am not sure about this. As it stands, mensural notation is indicated by the clef

Reply:
Sorry -- I meant clef. I'll fix this.